appserver/transaction/jts/src/main/java/com/sun/jts/codegen/jtsxa/OTSResourceHelper.java

```diff
@@ -53,13 +60,19 @@
 abstract public class OTSResourceHelper
 {
   private static String  _id = "IDL:jtsxa/OTSResource:1.0";
+  private static final Logger LOGGER = Logger.getLogger(OTSResourceHelper.class.getName());
 
   public static void insert (org.omg.CORBA.Any a, com.sun.jts.codegen.jtsxa.OTSResource that)
   {
-    org.omg.CORBA.portable.OutputStream out = a.create_output_stream ();
-    a.type (type ());
-    write (out, that);
-    a.read_value (out.create_input_stream (), type ());
+    try (org.omg.CORBA.portable.OutputStream out = a.create_output_stream ()) {
```

Comment:
Are you sure these are output and input streams that need closing here?

Reply:
Closing those streams will release ByteBuffers acquired more quickly. Will probably reduce memory footprint.

Reply:
I think closing this looks like a positive change, so I'm happy with it. Obviously it still needs a second reviewer though, so ping!